placekey/sf/doc/REFERENCE.md

```diff
@@ -0,0 +1,104 @@
+## placekey
+
+[Placekey](https://www.placekey.io/faq) is a free and open universal standard identifier for any physical place, so that the data pertaining to those places can be shared across organizations easily. Since Placekey is based on H3, here we offer a way to transform to and from that index and delegate any extra functionality to H3 itself.
+
+You can learn more about Placekey on [their website](https://www.placekey.io/) or in the [Overview section](../../overview/spatial-indexes/#placekey) of this documentation.
+
+### H3_ASPLACEKEY
+
+{{% bannerNote type="code" %}}
+PLACEKEY.H3_ASPLACEKEY(h3index)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns the placekey equivalent to the given H3 index.
+
+* `h3index`: `STRING` H3 identifier.
+
+**Return type**
+
+`STRING`
+
+**Example**
+
+```sql
+SELECT SFCARTO.PLACEKEY.H3_ASPLACEKEY('847b59dffffffff');
```

Comment:
I would use lowercase for the project name and module name to be aligned with the BigQuery reference. Also we should use `"` for the strings in the examples too.

```sql
SELECT sfcarto.placekey.H3_ASPLACEKEY("847b59dffffffff");
```

Reply:
yep, I remember. Actually, I was planning to do a pass on all the modules to update the doc, accordingly. Will update this one and create a separate PR to sync docs, READMEs and some other little changes in the rest of the modules